models/offer.py

```diff
@@ -116,34 +116,26 @@ def update_with_product_data(self, product_dict: dict):
         if owning_offerer and owning_offerer == self.venue.managingOfferer:
             self.product.populate_from_dict(product_dict)
 
+    @property
+    def activeMediation(self):
+        sorted_by_date_asc = sorted(self.mediations, key=lambda m: m.dateCreated)
```

Comment:
Les lignes 121 et 122 peuvent être fusionnées :

```suggestion
        sorted_by_date_asc = sorted(self.mediations, key=lambda m: m.dateCreated, reverse=True)
```

Reply:
En effet, merci :ok_hand: